libnmstate/ifaces/base_iface.py

```diff
@@ -327,6 +328,8 @@ def state_for_verify(self):
         _remove_lldp_neighbors(state)
         if Interface.STATE not in state:
             state[Interface.STATE] = InterfaceState.UP
+        if self.is_absent and not self._save_to_disk:
+            state[Interface.STATE] = InterfaceState.ABSENT
```

Comment:
Also need to the same to `pre_edit_validation_and_cleanup()`.

Reply:
Done, thanks!